### ISSUE TITLE
Duplicate menu key

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#27]
```
As of revision 93, the new sort menu feature was added.

Unfortunately, it's shortcut key is t, which conflicted with the previously 
existing toggle enabled, 
which is still t. However, it is now inaccessible due to the conflict, and the 
higher ordered 'sort' 
shortcut being defined first.

http://code.google.com/p/csshx/source/detail?r=93&path=/trunk/csshX#

The first two diffs tell the tale.
```

Original issue reported on code.google.com by `vxjaso...@gmail.com` on 8 Apr 2010 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
Thanks for spotting this. I guess it was bound to happen as features get added. 

Looks like "o" is available - I'll change it to that.

Committed to SVN (v0.64-104)
```

Original comment by `gavin.brock` on 9 Apr 2010 at 1:46
- Changed state: **Fixed**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `gavin.brock` on 9 Apr 2010 at 1:47
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I don't think you meant to change this back to Started :).
```

Original comment by `vxjaso...@gmail.com` on 22 Apr 2010 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
Thanks for checking - I was going to leave it as started (i.e. open) until I 
actually put it out as a release with the 
fix in it.

I guess I should really costomise the status types and make a more relavent one.
```

Original comment by `gavin.brock` on 22 Apr 2010 at 10:49

**Comment by GoogleCodeExporter** [comment]
```
Released in 0.65 and 0.70 releases.
```

Original comment by `gavin.brock` on 23 Apr 2010 at 3:51
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
Sorry - major bug in 0.70, please try 0.71
```

Original comment by `gavin.brock` on 25 Apr 2010 at 2:25
